docs/release_notes/v1.0.0-rc.2.md

```diff
@@ -0,0 +1,513 @@
+
+- **UPDATE** custom certs instructions [1008](https://github.com/dapr/docs/issues/1008)
+
+### Quickstarts
+
+- **UPDATED** observability quickstart to add instruction for self-host mode [288](https://github.com/dapr/quickstarts/issues/288)
+
+- **ADD** Support for ARM and ARM64 containers in quickstarts [327](https://github.com/dapr/quickstarts/issues/327)
+
+- **RESOLVED** update tracing config fields [335](https://github.com/dapr/quickstarts/pull/335)
+
+- **RESOLVED** Update quickstarts to not use exporter components [336](https://github.com/dapr/quickstarts/issues/336)
+
+### Samples
+
+- **RESOLVED** Add python version twitter processing pipeline app sample [36](https://github.com/dapr/samples/issues/36)
+
+### Test-infra
+
+- **ADD** CI/CD pipeline for longhaul tests [72](https://github.com/dapr/test-infra/issues/72)
+
+- **ADD** Create Prometheus dashboard for longhaul tests. [73](https://github.com/dapr/test-infra/issues/73)
+
+### Homebrew-tap
+
+- **ADD** RC releases for cli are now installable from Homebrew [11](https://github.com/dapr/homebrew-tap/issues/11)
+
+## Upgrading to Dapr 1.0.0-rc.2
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if docker command needs sudo:
+
+```bash
+dapr uninstall --all
+```
+
+For RC releases like this, download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have installed the CLI, run:
+
+```bash
+dapr init --runtime-version=1.0.0-rc.2
+```
+
+Wait for the update to finish,  ensure you are using the latest version of Dapr(1.0.0-rc.2) with:
+
+```bash
+$ dapr --version
+
+CLI version: 1.0.0-rc.3
+Runtime version: 1.0.0-rc.2
+```
+
+### Kubernetes
+
+#### Upgrading from previous version
+
+If you previously installed Dapr using Helm, you can upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh-install-on-a-cluster).
+
+##### 1. Get the latest CLI
+
+Download the latest RC release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+##### 2. Upgrade from existing cluster running 0.11.x
+
+Follow instructions below to migrate from 0.11.x placement service to 1.0.0-rc.2 placement service safely.
+
+First run these two commands to disable `helm upgrade` from uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement helm.sh/resource-policy=keep -n dapr-system
+kubectl annotate svc dapr-placement helm.sh/resource-policy=keep -n dapr-system
+```
+
```

Comment:
```suggestion

Now, check if you have any exporter type components, by running the following command `kubectl get components -o=custom-columns='NAME:metadata.name,TYPE:spec.type'` and look for components with type `exporter.*`.

Since exporter components are no longer supported, you will need to remove them prior to upgrading. You may want to print their content for your reference using `kubectl get component <component-name> -o yaml`.

```